holoviews/element/raster.py

```diff
@@ -408,45 +211,157 @@ class Image(SheetCoordinateSystem, Raster):
     access to the data, via the .data attribute.
     """
 
-    bounds = param.ClassSelector(class_=BoundingRegion, default=BoundingBox(), doc="""
+    bounds = param.ClassSelector(class_=BoundingRegion, default=None, doc="""
```

Comment:
Where does the assumption of unit bounds now come from? I'll edit this comment if I spot it later...